app/views/content/blog/returning-to-teaching-with-international-experience.md

```diff
@@ -66,4 +66,4 @@ My advice would be to:
     Ofsted is the [Office for Standards in Education, Children’s Services and Skills](https://www.gov.uk/government/organisations/ofsted/about) that inspects services providing learning and education.
     The [Ofsted education inspection framework](https://www.gov.uk/government/publications/education-inspection-framework) sets out how they inspect schools in England.
 
-We’re here to answer your questions and provide advice about teaching, whether you’re just thinking about it or you’re ready to apply. [Find out how you can get help and support with getting into teaching](/help-and-support).
+We’re here to answer your questions and provide advice about teaching, whether you’re just thinking about it or you’re ready to apply. [Find out how you can get help and support with getting into teaching](/help-and-support).
```

Comment:
Thanks for reinstating this blog @philspray - just need to remove the `returning to teaching` from it now

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/e22f3ddd-8e38-4013-b56f-2a109ee9199e)